lib/bolt/cli.rb

```diff
@@ -291,6 +291,16 @@ def validate(options)
         raise Bolt::CLIError, "Must specify a plan name."
       end
 
+      if options[:subcommand] == 'module' && options[:action] == 'add'
+        unless options[:object]
+          raise Bolt::CLIError, "Must specify a module name."
+        end
+
+        if options[:leftovers].any?
```

Comment:
It looks like this is never hit because we error on line 241 first:
```
lucy ~/githubs/bolt ((36c8295...)) $ be bolt module add puppetlabs-apache puppetlabs-aws_inventory --project project/
Unknown argument(s) puppetlabs-aws_inventory
```

You could just add an `elsif subcommand == 'module'` there, or leave that message as is and remove this clause.

Reply:
Er, not sure how I missed that...